data/core/encyclopedia/geography.cfg

```diff
@@ -233,12 +246,13 @@ Yet despite the sheer wilderness above the Northlands, the subterranean Dwarvish
 [topic]
     id=far_south
     title= _ "Far South"
-    text= _ "A vast region south of Kerlath home to the Dunefolk Civilization. It is also inhabited by many local creatures, including the Jinn and Wyverns." + "
+    text= _ "A vast region south of Kerlath home to many mysterious races, including the Jinn and Wyverns. Very little is known for certain about what lies here." + "
 
 " +
         #po: Geography of the Far South
         _ "<header>text='Notable land features:'</header>
-    • Sandy Wastes: A hot desert located south of Kerlath home to various Dunefolk settlements, as well as nomadic tribes in between. The northern parts of this desert are home to many treacherous monsters.
-    • Mountains of Peril: A tall and deadly mountain range in the far south inhabited by bands of vicious barbarians and monsters.
-    • Black Forest: An ancient and likely haunted forest of which very little is known, believed to have been once inhabited by elves. Those who enter it seldom return."
+    • Mountains of Peril: A tall and deadly mountain range in the Far South inhabited by bands of vicious barbarians and monsters.
+    • Sandy Wastes: A hot and dry desert located southeast of Kerlath. It is considered an impassable barrier by Wesnothians, filled with very sparse vegetation consisting of no more than a few shrubs and cacti.
```

Comment:
Isn't a desert dry by definition? You can have a cold desert, but not a non-dry one, I don't think.